templates/data/_partials/value_details.html

```diff
@@ -1,13 +1,39 @@
 {% load custom_filters %}
 
-{% if value|valuetype != "list" and value|valuetype != "dict" %}
+{# Recursion on lists and dicts #}
+{% if value|valuetype == "list" and value %}
+    <ul>
+        {% for item in value %}
+            <li>{% include "data/_partials/value_details.html" with key=key value=item %}</li>
+        {% endfor %}
+    </ul>
+{% elif value|valuetype == "dict" %}
+        {% for key, item in value.items %}
+            <p><strong>{{ key }}</strong> : {% include "data/_partials/value_details.html" with key=key value=item %}</p>
+        {% endfor %}
+
+        {% comment %}
+            If the value is a dict containing "adresse", "ville" and "code_postal",
+            we can display a link to Google Maps AFTER recusrively displaying the rest,
+            this works both in suggestion changes and contexte
+        {% endcomment %}
+        {% if "adresse" in value and "ville" in value and "code_postal" in value %}
+            <br>
+            {# As bonus we add "nom" to the query if present #}
+            <a href="https://www.google.com/maps/search/?api=1&query={{ value.nom|urlencode }}+{{ value.adresse|urlencode }}+{{ value.code_postal|urlencode }}+{{ value.ville|urlencode }}" target="_blank">
```

Comment:
```suggestion
            <a href="https://www.google.com/maps/search/?api=1&query={{ value.nom|urlencode }}+{{ value.adresse|urlencode }}+{{ value.code_postal|urlencode }}+{{ value.ville|urlencode }}" target="_blank" rel="noreferrer noopener">
```

Mesures de sécurité 
Le noopener est facultatif sur la plupart des navigateurs récents

Reply:
Aussi, je serais d'avis de construire l'url en python plutôt que dans le html